### ISSUE TITLE
Replace hard-coded dimensions by dimen resource values

### ISSUE THREAD

**Issue by arundhati24** [nextcloud/android#2358]
### Actual behaviour

Currently, there are a number of hard-coded dimensions present in the layout resource files, which is not considered a good practice.

### Expected behaviour

Dimensions should be referenced from dimens.xml rather than hard-coding them directly into the code.

**Comment by tobiasKaminsky** [comment]
:+1: Fine by me.
There are a lot of dimensions already, but if a value is missing, simply create it with a sane name.
(please wait for a second approval before starting it)

**Comment by arundhati24** [comment]
@tobiasKaminsky Sure. I found some hard-coded dimensions while working on another issue. But you're right, most of the dimensions are already referenced from dimens.xml :)

**Comment by mario** [comment]
Sure.

**Comment by AndyScherzinger** [comment]
Sure 👍
